orderbook/src/fee.rs

```diff
@@ -168,7 +168,7 @@ impl MinFeeCalculator {
         amount: Option<U256>,
         kind: Option<OrderKind>,
     ) -> Result<U256, PriceEstimationError> {
-        let gas_price = self.gas_estimator.estimate().await?;
+        let gas_price = self.gas_estimator.estimate().await?.effective_gas_price();
```

Comment:
This is a gotcha! We want the effective gas price for fee computations and not gas caps.